src/loader/bgw_interface.c

```diff
@@ -23,6 +23,8 @@ TS_FUNCTION_INFO_V1(ts_bgw_db_workers_stop);
 
 TS_FUNCTION_INFO_V1(ts_bgw_db_workers_restart);
 
+TS_FUNCTION_INFO_V1(ts_bgw_db_workers_force_restart);
```

Comment:
I think we should be consistent about start_or_restart or force_restart. I prefer the former